src/ngraph_builder.cc

```diff
@@ -1612,11 +1575,10 @@ tf::Status Builder::TranslateGraph(const std::vector<tf::TensorShape>& inputs,
     // Pack
     // -------
     else if (op->type_string() == "Pack") {
-
-      std::cout << "Number of inputs : " << op->num_inputs() << std::endl;
       if (op->num_inputs() < 1) {
         return tf::errors::InvalidArgument(
-            "Number of inputs should be at least 1 for Pack");
+            op->name(), " requires at least 1 input, got ", op->num_inputs(),
```

Comment:
The same new suggested validation function in line 565 may be used here as well.